### ISSUE TITLE
Stellar Aphostrophy" by Doubly Quoter's

### ISSUE THREAD

**Issue by comradekingu** [airsonic/airsonic#645]
Instances of un-escaped double quotation marks:
["admin"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L34)
["{0}"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L124)
["{0}"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L125)
["{0}"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L381)
[(")](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L660)
["The" and "#"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L680)
["The"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L682)
["Sound tracks"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L682)
["Work" and "Living room"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L717)
["Play"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L718)
["None"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L730)

- [ ] 
---

Missing double quotation marks:
["Home" button](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L155)
["Getting started"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L164)
[Click "Help"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L466)
[Click "Legacy""Embedded JDBC" "JNDI"](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L656)

---
Checkem' The double singles club.
[2x](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L423)
[Apple''s](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L423)
[2x](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L704)
[doesn''t](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L712)
[don't](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L714)

---

Bonus oddments:
Why aren't these ", but instead '' ?
[](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L64)  
["window.open(''https://airsonic.github.io/download/'')">here</a>.](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L272) 
[same thing)">here</a>.](https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L64)  

----
The idea is to establish a convention, group by linenumber, and get it fixed.
HTML "&apos;" and &quot;
Java ???
C  "\'" and "\""

**Comment by jooola** [comment]
Something is wrong in the album dropdown menu ( "selected songs" or so ) down the page

**Comment by muff1nman** [comment]
Fixed that @jooola  in a93a18a055aa90d5439ce87aa8f1bce12fdc7d84

**Comment by heyarne** [comment]
I was told to report this here. I've got funky characters showing up after a fresh install of the current release version:

<img width="512" alt="image" src="https://user-images.githubusercontent.com/525780/34248213-79dda850-e635-11e7-8389-38c4adc0c1ad.png">
<img width="512" alt="image" src="https://user-images.githubusercontent.com/525780/34248217-7d55f2b2-e635-11e7-8fa6-1fc875f306c4.png">

**Comment by comradekingu** [comment]
@heyarne that is the typographically correct ellipsis … as opposed to a tripledot ...
It got changed here https://github.com/airsonic/airsonic/pull/643/files#diff-c6a2cdcb222406336c6ab8c296cf0a07L552
But is still tripledot https://github.com/airsonic/airsonic/blob/dbd2a738eb0fe84d7d73191c2f2b79aa0afa6c59/airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_en.properties#L88

.../…? indeed

**Comment by muff1nman** [comment]
Yesterday I pushed a change to go back to triple dot, but looking at it some more I think the better fix would be to change instances like this:

```
<%@ page language="java" contentType="text/html; charset=utf-8" pageEncoding="iso-8859-1"%>
```
So that `pageEncoding` is utf8. My hunch is that should fix problems like these. Can someone take that task?

**Comment by jooola** [comment]
@muff1nman If this is a stupid Search&Replace I can do it

**Comment by muff1nman** [comment]
search/replace and test ;) basically we want to ensure that ` <meta charset="ISO-8859-1"> ` doesn't show up on any html output. If we do have a `meta` tag it should be utf8.

**Comment by jooola** [comment]
See #660

**Comment by wildcard418** [comment]
This still seems to be an issue on version 10.1.2-RELEASE – July 28, 2018 standalone.

![image](https://user-images.githubusercontent.com/46544275/50934853-935ee900-142f-11e9-8c19-dd83ec5f1379.png)

**Comment by urgaylol** [comment]
Can confirm, it is still an issue on the latest version. (10.1.2)

**Comment by jvoisin** [comment]
We still have some `ISO-8859-1` occurrences in the codebase:

```
$ git grep 'ISO-8859-1'
airsonic-main/src/main/java/org/airsonic/player/taglib/UrlTag.java:51: * but most(?) servlet container assumes that ISO-8859-1 is used.)
airsonic-main/src/main/java/org/airsonic/player/util/StringUtil.java:44:    public static final String ENCODING_LATIN = "ISO-8859-1";
airsonic-main/src/main/resources/applicationContext-cache.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
airsonic-main/src/main/resources/applicationContext-service.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
airsonic-main/src/main/resources/applicationContext-sonos.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
airsonic-main/src/main/resources/servlet.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
airsonic-main/src/main/webapp/WEB-INF/dwr.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
airsonic-main/src/main/webapp/WEB-INF/sub.tld:1:<?xml version="1.0" encoding="ISO-8859-1" ?>
airsonic-main/src/main/webapp/WEB-INF/sub.tld:42:            <description>The encoding to use. Default is ISO-8859-1.</description>
airsonic-main/src/test/resources/applicationContext-mockSonos.xml:1:<?xml version="1.0" encoding="ISO-8859-1"?>
pom.xml:245:                    <encoding>ISO-8859-1</encoding>
$
```

**Comment by jooola** [comment]
@jvoisin the Search & Replace in #660 hasn't been merged.

**Comment by jvoisin** [comment]
Yeah, it seems that #674 was merged instead :/
I'm all for converting the rest of `ISO-8859-1` to UTF-8. Do you want to do it, or shall I open a PR?

**Comment by jooola** [comment]
Feel free to open a new one.

**Comment by jvoisin** [comment]
Done in #983

**Comment by jvoisin** [comment]
Can anyone check that this fixes the issue?

**Comment by kaysond** [comment]
Appears to be fixed on 10.3.1

**Comment by jvoisin** [comment]
Wonderful, thank you for testing this ♥
